sites/public/cypress/e2e/pages/listings-map.spec.ts

```diff
@@ -77,20 +77,6 @@ describe.skip("Listings map", function () {
     cy.get("@markersSearch.all").should("have.length", 3)
     cy.get("@listingsSearch.all").should("have.length", 4)
 
-    // Click manually into empty space
-    cy.getByTestId("map").dblclick(300, 20)
```

Comment:
I can't comment on the actual line (line 73) but one issue that has come up a couple of times is that clicking on a marker the `listings-map-info-window` appears but the data doesn't populate. Is there a way we could verify that some data is there? I know that we can't rely on the listing being the same every time but could we verify that the "see details" button is there?

Reply:
Yes probably! I'll take a look!

Reply:
Just added this extra assert 👍